WATOLINK/bci-gaming
# Ryan win sequence

## ClickUp ticket link

[Game - Winning Sequence](https://app.clickup.com/t/860q9n3kv)



## Implementation description
Created art for a screen that shows after completing a level and implemented in Unity.
Currently, the next level and home buttons are functional. The other buttons have placeholder scripts for now.



## Steps to test
1. Open the Win Sequence Scene
2. Ensure all components are visible
3. Run and test all buttons



## What should reviewers focus on?
Ensure the new scene is functional


## Checklist
